config/jobs/kubernetes/sig-k8s-infra/registry.k8s.io/canaries.yaml

```diff
@@ -106,7 +106,7 @@ periodics:
           -v 2 \
           --up --down \
           --cloud-provider=aws \
-          --create-args="--channel=alpha --networking=kubenet --container-runtime=containerd --set=spec.assets.containerProxy=registry-sandbox.k8s.io --set=cluster.spec.cloudProvider.aws.nodeTerminationHandler.enabled=false --discovery-store=s3://k8s-kops-prow/discovery" \
+          --create-args="--channel=alpha --networking=kubenet --container-runtime=containerd --set=spec.assets.containerProxy=registry-sandbox.k8s.io --set=cluster.spec.cloudProvider.aws.nodeTerminationHandler.enabled=false --set=cluster.spec.cloudConfig.awsEBSCSIDriver.enabled=false --discovery-store=s3://k8s-kops-prow/discovery" \
```

Comment:
`awsEBSCSIDriver.enabled` may need to be changed to `cloudProvider.aws.ebsCSIDriver`.
See:
https://github.com/kubernetes/kops/blob/41480d95a96edad247ecd777b2790ecda33f6a85/docs/cluster_spec.md?plain=1#L1555